qiskit/optimization/algorithms/grover_optimizer.py

```diff
@@ -39,23 +39,39 @@ class GroverOptimizer(OptimizationAlgorithm):
     """Uses Grover Adaptive Search (GAS) to find the minimum of a QUBO function."""
 
     def __init__(self, num_value_qubits: int, num_iterations: int = 3,
-                 quantum_instance: Optional[Union[BaseBackend, QuantumInstance]] = None) -> None:
+                 quantum_instance: Optional[Union[BaseBackend, QuantumInstance]] = None,
+                 converters: Optional[Union[QuadraticProgramConverter,
+                                            List[QuadraticProgramConverter]]] = None) -> None:
         """
         Args:
             num_value_qubits: The number of value qubits.
             num_iterations: The number of iterations the algorithm will search with
                 no improvement.
             quantum_instance: Instance of selected backend, defaults to Aer's statevector simulator.
+            converters: The converters to use for converting a problem into a different form.
+                If not specified, ``QuadraticProgramToQubo`` will be used.
```

Comment:
```suggestion
                By default, when None is specified, an internally created instance of
                :class:`~qiskit.optimization.converters.QuadraticProgramToQubo` will be used.
```
By using :class: this will have Sphinx create a link in the html which is much more convenient for a user to be able to navigate to this rather than just having highlighted text